client/src/app/pages/tasks/tasks-page.tsx

```diff
@@ -0,0 +1,348 @@
+      },
+      {
+        categoryKey: "state",
+        title: t("terms.status"),
+        type: FilterType.search,
+        placeholderText: t("actions.filterBy", {
+          what: t("terms.status") + "...",
+        }),
+        getServerFilterValue: (value) => (value ? [`*${value[0]}*`] : []),
+      },
+      {
+        categoryKey: "application",
+        title: t("terms.application"),
+        type: FilterType.search,
+        placeholderText: t("actions.filterBy", {
+          what: t("terms.application") + "...",
+        }),
+        serverFilterField: "application.name",
+        getServerFilterValue: (value) => (value ? [`*${value[0]}*`] : []),
+      },
+      {
+        categoryKey: "kind",
+        title: t("terms.kind"),
+        type: FilterType.search,
+        placeholderText: t("actions.filterBy", {
+          what: t("terms.kind") + "...",
+        }),
+        getServerFilterValue: (value) => (value ? [`*${value[0]}*`] : []),
+      },
+      {
+        categoryKey: "createUser",
+        title: t("terms.createdBy"),
+        type: FilterType.search,
+        placeholderText: t("actions.filterBy", {
+          what: t("terms.createdBy") + "...",
+        }),
+        getServerFilterValue: (value) => (value ? [`*${value[0]}*`] : []),
+      },
+    ],
+    initialItemsPerPage: 10,
+  });
+
+  const {
+    result: { data: currentPageItems = [], total: totalItemCount },
+    isFetching,
+    fetchError,
+  } = useServerTasks(
+    getHubRequestParams({
+      ...tableControlState,
+      hubSortFieldKeys: {
+        id: "id",
+        state: "state",
+        application: "application.name",
+        kind: "kind",
+        createUser: "createUser",
+        priority: "priority",
+      },
+    })
+  );
+
+  const tableControls = useTableControlProps({
+    ...tableControlState,
+    // task.id is defined as optional
+    idProperty: "name",
+    currentPageItems,
+    totalItemCount,
+    isLoading: isFetching,
+    selectionState: useSelectionState({
+      items: currentPageItems,
+      isEqual: (a, b) => a.name === b.name,
+    }),
+  });
+
+  const {
+    numRenderedColumns,
+    propHelpers: {
+      toolbarProps,
+      filterToolbarProps,
+      paginationToolbarItemProps,
+      paginationProps,
+      tableProps,
+      getThProps,
+      getTrProps,
+      getTdProps,
+      getColumnVisibility,
+    },
+    columnState,
+  } = tableControls;
+
+  const clearFilters = () => {
+    const currentPath = history.location.pathname;
+    const newSearch = new URLSearchParams(history.location.search);
+    newSearch.delete("filters");
+    history.push(`${currentPath}`);
+    filterToolbarProps.setFilterValues({});
+  };
+
+  const { cancelTask, togglePreemption } = useTaskActions();
+
+  const toCells = ({
+    id,
+    application,
+    kind,
+    state,
+    priority = 0,
+    policy,
+    createUser,
+    pod,
+    started,
+    terminated,
+  }: Task) => ({
+    id,
+    application: application.name,
+    kind,
```

Comment:
To best support old tasks, this should probably be `kind: kind ?? addon`

#1970 / #1959 needs to be merged before analyzer tasks will be created with the kind set instead of addon